parquet-column/src/test/java/org/apache/parquet/schema/TestPrimitiveStringifier.java

```diff
@@ -165,6 +167,7 @@ public void testTimestampMillisStringifier() {
     PrimitiveStringifier stringifier = PrimitiveStringifier.TIMESTAMP_MILLIS_STRINGIFIER;
 
     assertEquals("1970-01-01T00:00:00.000", stringifier.stringify(0l));
+    assertEquals("1970-01-01T00:00:00.000+0000", PrimitiveStringifier.TIMESTAMP_MILLIS_UTC_STRINGIFIER.stringify(0l));
```

Comment:
Could you make this more consistent? Currently one stringifier is in a local variable, while the other is referenced directly.